Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/manager/RequestsManager.java

```diff
@@ -113,4 +113,9 @@ protected String generateId(String prefix) {
     public List<ReviewRequest> getAllReviews() {
        return dbClient.getAllReviews();
     }
+
+    @Override
+    public List<ReviewRequest> getAllOpenReviews() {
```

Comment:
Just make that return reviews by status and then call it whereever you need all reviews with a specific status.